client-reactjs/src/components/application/Assets/AssetsTable.js

```diff
@@ -220,6 +222,7 @@ function AssetsTable({ selectedGroup, openGroup, handleEditGroup, refreshGroups
           refreshGroups();
         }
         message.success(type + " deleted sucessfully");
+        fetchDataAndRenderTable();
```

Comment:
Looks like this call is happening a few lines above this. Are both calls required?

Reply:
It is not required. I have removed it. Not sure how it got there.